src/main/scala/com/fulcrumgenomics/umi/DuplexConsensusCaller.scala

```diff
@@ -216,6 +219,9 @@ class DuplexConsensusCaller(override val readNamePrefix: String,
     }
   }
 
+  /** Extracts the UMI bases for each source read. */
+  private def toUmiBases(reads: Seq[SourceRead]): Seq[String] = reads.flatMap(_.sam).flatMap(_.get[String](ConsensusTags.UmiBases))
```

Comment:
Is the use of `.get` and `.flatMap` here to handle the case that we're presented a BAM that doesn't have `RX`?  If so, maybe just note that so we don't forget?